numpy/meson.build

```diff
@@ -119,7 +144,16 @@ if have_blas
       #define EXPAND(suffix) cblas_ddot ## suffix
       #define DDOT(suffix) EXPAND(suffix)
 
-      #include <cblas.h>
+      #if defined __has_include
+        #if __has_include(<Accelerate/Accelerate.h>)
```

Comment:
This won't be right if we're building against a library other than Accelerate - it'll include `Accelerate.h` rather than `cblas.h` for this check.